cmake/branding/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+CONFIGURE_FILE("recolor_graphics.sh.in" "recolor_graphics.sh" @ONLY)
+FILE(COPY svg/ DESTINATION svg/)
+# FIXME, this runs each time cmake is called and it's a tad slow
+EXECUTE_PROCESS(COMMAND "${CMAKE_CURRENT_BINARY_DIR}/recolor_graphics.sh")
```

Comment:
CMake will fail if this command fails. Since the shell script assumes `rsvg-convert` is available, we should check for `rsvg-convert` either here or in the script itself.

Reply:
Also, quoting the CMake documentation for `execute_process`:
> The execute_process() command is a newer more powerful version of exec_program(), but the old command has been kept for compatibility. Both commands run while CMake is processing the project prior to build system generation. Use add_custom_target() and add_custom_command() to create custom commands that run at build time.

Reply:
> CMake will fail if this command fails

Fixed first part per 7ffac7a

Reply:
Second part addressed via 8ffca1e.  `add_custom_target` really stinks unless you can bind to a build task and branding can occur at any stage.  For now I'm relying on `LMMS_RELEASE_TYPE` not changing to prevent it from running again and again on subsequent builds.